test/helpers/generators.go

```diff
@@ -139,6 +141,20 @@ func GenerateGatewayConfiguration(namespace string) *operatorv1beta1.GatewayConf
 			},
 		},
 	}
+	for _, opt := range opts {
+		opt(gwc)
+	}
+	return gwc
+}
+
+// WithWebhookDisabled disables the admission webhook for the control plane
+func WithWebhookDisabled() func(*operatorv1beta1.GatewayConfiguration) {
```

Comment:
I'd consider calling this `WithControlPlaneWebhookDisabled` just to be a bit more explicit, WDYT?